Topics/User_Experience.md

```diff
@@ -1,48 +1,206 @@
 
-In addition to Bootstrap, Ant Design is another popular UI library and design system. It is specifically designed for React applications and other frontend frameworks like Vue, Angular and etc, providing a complete set of high-quality React components that make user interface development simpler, more consistent, and modular. Ant Design is ideal for building enterprise-level applications and dashboards with a focus on high performance, usability, and aesthetics.
+# Tools Used in User Experience Design
+## Design Tools
+### Figma
+
+Figma is a free (for the most part) collaborative tool that lets you design and build mock-ups and prototypes of your software/applications. You may not need to build prototypes as a software developer/engineer, but you'll probably find yourself viewing Figma design files to build your frontend. So, now's a great time to start learning how to use [Figma](https://www,figma.com)!
+
+**Helpful tip:** With just a click of a button, you can switch to "developer mode" in Figma, which helps you translate design into code! Essentially, you can click on components like a button and get its corresponding code and styling. Read about how to use this feature [here](https://www.figma.com/dev-mode/).
+
+**Figma, MUI, and React:** See the [UI Component Libraries](https://learning-software-engineering.github.io/Topics/User_Experience.html#ui-component-libraries) section for details on MUI. Note that MUI is a component library for React _only_. You can download the** MUI extension on Figma** and immediately use MUI components in your Figma design. Then, when you're ready to code, you can install MUI in your React project and easily create your frontend by entering "developer mode" on your Figma design and copying the code that Figma generated for you.
+
+### Adobe XD
+
+[Adobe XD](https://www.adobe.com/products/xd/learn/get-started/what-is-adobe-xd-used-for.html) is another popular prototyping tool. Unfortunately, the current author of this wiki doesn't know much about it, other than it requires a monthly subscription :(.
+
+## UI Component Libraries
+When building an application with a frontend framework (e.g., React, Vue, Angular, etc.), developers can import an UI design component library/libraries to help them easily build aesthetic, user-friendly interfaces. 
+
+### Material UI (MUI)
+
+[MUI](https://mui.com/) is a popular library of UI design components made for React. These components are essentially pre-made "features" that you can immediately use, such as buttons, navbars, pop-ups, etc.
+
+### Ant Design
+
```

Comment:
Maybe consider having ANT Design as a new page that is linked to this main page, just because it has all the components (summary, how-to, additional links) for a separate page on the wiki.

Reply:
Thanks for the suggestion! I think that'd a good call since this User Experience document is getting a little long.